modules/source/mail/config/mail.xml

```diff
@@ -34,7 +38,7 @@
 			store-uri="${protocol}://${username:}:${password:}@${host}:${port}/${folder}"
```

Comment:
I noticed there are few more JavaMail properties that user would like to add based on the need. In that case, they would need to edit the module xml to add new properties. I thought having them in config home could help. Also, once we have mail sink module updated for the same, we can add `SMTP` config there as well.